shell/platform/fuchsia/dart_runner/dart_test_component_controller_v2.cc

```diff
@@ -377,7 +378,7 @@ void DartTestComponentControllerV2::CaseIterator::GetNext(
   // test case. Flip flag once the one test case has been retrieved.
   if (first_case_) {
     fuchsia::test::Case test_case;
-    test_case.set_name(std::string(kTestCaseName));
+    test_case.set_name(test_component_name_);
```

Comment:
Eventually we'll want to change this so it's the name of the actual test case instead of the name of the component right? Should we leave a TODO here for that?

Reply:
That is a "maybe". Right now, if I can wire the output such that it displays to stdout, instead of `fx log`, there won't be a need for that, since the stdout sort of already displays each test name and exit. To be able to pass each individual test name to a test_case will be a little more involved. I can explain in 1:1 .